src/pages/Travel/MyTripsPage.tsx

```diff
@@ -21,7 +21,7 @@ function MyTripsPage() {
         >
             <FullPageNotFoundView
                 shouldForceFullScreen
-                shouldShow={(!canUseSpotnanaTravel && !NativeModules.HybridAppModule) || (isBlockedFromSpotnanaTravel && !NativeModules.HybridAppModule)}
+                shouldShow={(!canUseSpotnanaTravel && !CONFIG.IS_HYBRID_APP) || (isBlockedFromSpotnanaTravel && !CONFIG.IS_HYBRID_APP)}
```

Comment:
Maybe we could simplify it slightly? NAB, though 😄 
```suggestion
                shouldShow={!CONFIG.IS_HYBRID_APP && (isBlockedFromSpotnanaTravel || !canUseSpotnanaTravel)}
```